flower_shop/stylles/general.css

```diff
@@ -0,0 +1,25 @@
+@font-face {
+  font-family: miltonian;
+  src: url(../fonts/Miltonian/Miltonian-Regular.ttf);
+}
+@font-face {
+  font-family: monda;
+  src: url(../fonts/monda/Monda-Regular.ttf);
+}
+@font-face {
+  font-family: montserrat;
+  src: url(../fonts/Montserrat/Montserrat-Italic-VariableFont_wght.ttf);
+}
+body {
+  margin: 0;
+}
+:root {
```

Comment:
in home task you had to create also the same variables but for other theme

so home task is not completely done